mocks/organisasjon-mock/src/main/java/no/nav/tjeneste/virksomhet/organisasjon/rs/OrganisasjonAdresse.java

```diff
@@ -0,0 +1,149 @@
+        } else {
+            this.organisasjonDetaljer.registreringsdato = LocalDateTime.now().minusYears(1);
+        }
+        if (modell.getOrganisasjonDetaljer() != null && modell.getOrganisasjonDetaljer().getForretningsadresser() != null){
+            this.organisasjonDetaljer.forretningsadresser = modell.getOrganisasjonDetaljer().getForretningsadresser();
+        }
+        if (modell.getOrganisasjonDetaljer() != null && modell.getOrganisasjonDetaljer().getPostadresser() != null){
+            this.organisasjonDetaljer.postadresser = modell.getOrganisasjonDetaljer().getPostadresser();
+        }
+    }
+
+    public String getOrganisasjonsnummer() {
+        return this.organisasjonsnummer;
+    }
+
+    public OrganisasjonstypeEReg getType() {
+        return this.type;
+    }
+
+    public String getNavn() {
+        return this.navn != null ? this.navn.getNavn() : null;
+    }
+
+    public AdresseEReg getKorrespondanseadresse() {
+        return !this.getPostadresser().isEmpty() ? (AdresseEReg)this.getPostadresser().get(0) : (AdresseEReg)this.getForretningsadresser().get(0);
+    }
+
+    public List<AdresseEReg> getForretningsadresser() {
+        return this.organisasjonDetaljer != null ? this.organisasjonDetaljer.getForretningsadresser() : Collections.emptyList();
+    }
+
+    public List<AdresseEReg> getPostadresser() {
+        return this.organisasjonDetaljer != null ? this.organisasjonDetaljer.getPostadresser() : Collections.emptyList();
+    }
+
+    public LocalDate getRegistreringsdato() {
+        return this.organisasjonDetaljer != null && this.organisasjonDetaljer.getRegistreringsdato() != null ? this.organisasjonDetaljer.getRegistreringsdato().toLocalDate() : null;
+    }
+
+    public LocalDate getOpphørsdato() {
+        return this.organisasjonDetaljer != null ? this.organisasjonDetaljer.getOpphoersdato() : null;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    static class OrganisasjonDetaljer {
+        @JsonProperty("registreringsdato")
+        private LocalDateTime registreringsdato;
+        @JsonProperty("opphoersdato")
+        private LocalDate opphoersdato;
+        @JsonProperty("forretningsadresser")
+        private List<AdresseEReg> forretningsadresser;
+        @JsonProperty("postadresser")
+        private List<AdresseEReg> postadresser;
+
+        private OrganisasjonDetaljer() {
+        }
+
+        private LocalDateTime getRegistreringsdato() {
+            return this.registreringsdato;
+        }
+
+        private LocalDate getOpphoersdato() {
+            return this.opphoersdato;
+        }
+
+        private List<AdresseEReg> getForretningsadresser() {
+            return this.forretningsadresser != null ? this.forretningsadresser : Collections.emptyList();
+        }
+
+        private List<AdresseEReg> getPostadresser() {
+            return this.postadresser != null ? this.postadresser : Collections.emptyList();
+        }
+    }
+
+    static class Navn {
```

Comment:
Er alle disse navnelinjene obligatoriske eller skulle det legges til annotering @JsonIgnoreProperties(ignoreUnknown = true) for klassen?

Reply:
De er obligatoriske, så her bør man ikke ignorere unknown ettersom de alltid skal være satt. Kan ikke ha org uten navn.